script/generate-release-notes.js

```diff
@@ -1,4 +1,5 @@
-const octokit = require('@octokit/rest')()
```

Comment:
`tsc` got confused by this require statement when trying to mesh it with the included type declarations:

<img width="452"  src="https://user-images.githubusercontent.com/359239/50053493-999dc600-010c-11e9-989a-558c79ec15e4.png">


Breaking it out into two lines made this easier for it to follow, and also enables better use of the included type declarations (if required) later on.